### PR TITLE
Issue 70 request to add more button variant and create a button reference example page

### PR DESCRIPTION
**Pull request by SeiKasahara**
## Change Summary
Readded the button component in component-showcase page.
Added all button component variant include:

1. Inactive
2. Small One (default and cutout)
3. Default
4. Finish
5. Cutout
6. Link

### Change Form

- [x] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [x] The change has tests
- [ ] The change has documentation

### CODE REVIEW

File: client/src/components/ui/button.tsx [hunk 1]
```diff
@@ -5,29 +5,31 @@ import * as React from "react";
...
       size: {
         default: "h-10 px-4 py-2",
-        sm: "h-9 rounded-md px-3",
+        penni: "w-[343px] h-[54px]", // Penni Default Size
+        sm: "w-[132px] h-[38px]",
         lg: "h-11 rounded-md px-8",
         icon: "h-10 w-10",
+        floating: "w-[120px] h-[40px]",
+        pay: "w-[58px] h-[24px]",
+        finish: "w-[335px] h-[54px]",
```
Comment (SodaVolcano) [inline client/src/components/ui/button.tsx]: Change the relevant widths to `w-full` (e.g. `penni`).  If needed sizing should be done by wrapping the button inside a `<div>` with the specified width. Also please replace all styles with tailwind classes, and chose the closest class if no exact match exist. e.g. if the Figma style had `height: 17px`, select the tailwind class with `height: 16px` or `height: 18px`

File: client/src/components/ui/button.tsx [hunk 2]
```diff
@@ -5,29 +5,31 @@ import * as React from "react";
 import { cn } from "@/lib/utils";
 
 const hoverStyles =
-  "hover:bg-white hover:text-penni-main hover:border hover:border-2 hover:border-penni-main hover:rounded-lg hover:border-solid";
+  "hover:bg-white hover:text-penni-main hover:headline hover:border hover:border-2 hover:border-penni-main hover:rounded-lg hover:border-solid";
 // rounded-lg corresponds to 8px border radius
 // added hover variant
 const buttonVariants = cva(
-  "inline-flex items-center justify-center whitespace-nowrap rounded-lg text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 font-sans",
+  "inline-flex items-center justify-center whitespace-nowrap text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 font-sans",
```
Comment (SodaVolcano) [inline client/src/components/ui/button.tsx]: Remove `font-sans`, font is defined in `Styles.css`

File: client/src/components/ui/icons.tsx [hunk 3]
```diff
@@ -133,7 +133,7 @@ const MeIcon: React.FC<BottomNavIconProps> = ({ className, ...props }) => {
       <path
         d="M20 21v-2a4 4 0 00-4-4H8a4 4 0 00-4 4v2M12 11a4 4 0 100-8 4 4 0 000 8z"
         stroke="currentColor"
-        strokeWidth={2}
+        stroke-width={2}
```
Comment (SodaVolcano) [inline client/src/components/ui/icons.tsx]: Why were these changed to `stroke-width`? `StrokeWidth` should work?

File: client/tailwind.config.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import { finished } from "stream";
```
Comment (SodaVolcano) [inline client/tailwind.config.ts]: Why was this imported? By accident perhaps?

**Review by SodaVolcano — CHANGES_REQUESTED** [review]

File: client/src/components/ui/button.tsx (hunk 1 above)
Comment (SeiKasahara) [reply client/src/components/ui/button.tsx]: Now revised all widths and heights.

**Comment by SeiKasahara** [comment]
Inactivated Button now has been set as `disabled={true}`.

**Review by SodaVolcano — APPROVED** [review]
Looks good to me
